### ISSUE TITLE
TypeScript types need to be published

### ISSUE THREAD

**Issue by CruseCtrl** [crubier/react-graph-vis#136]
TypeScript types [were merged in](https://github.com/crubier/react-graph-vis/pull/113) a little while ago. Could these be published to NPM please? Thanks!

**Comment by Sheldonfrith** [comment]
Second this

**Comment by LorhanSohaky** [comment]
@crubier , can you help with this?

**Comment by AndrewGardhouse** [comment]
Any updates on this?

**Comment by acbart** [comment]
Yes, I'd love to use this library but typescript support is important!

**Comment by kassfir** [comment]
FYI, I discovered this packagage that solves the problem: 

https://www.npmjs.com/package/react-vis-graph-wrapper
